site/docs/install-ubuntu.md

```diff
@@ -22,6 +23,32 @@ Bazel comes with two completion scripts. After installing Bazel, you can:
 *   Access the [bash completion script](completion.md#bash)
 *   Install the [zsh completion script](completion.md#zsh)
 
+<h2 id="install-with-package-installer-ubuntu">Installing using package installer</h2>
+
+The debian package files are on Bazel's [GitHub releases page](https://github.com/bazelbuild/bazel/releases).
+
+### Step 1: Install required packages
```

Comment:
I would rather not have this. The .deb package should list it's dependencies and those should auto-install.
If we do not have the right set, let's fix that problem. The current control file is

Package: bazel
Version: 0.24.1
Section: contrib/devel
Priority: optional
Architecture: amd64
Depends: g++, zlib1g-dev, bash-completion, unzip, python
Suggests: google-jdk | java8-sdk-headless | java8-jdk | java8-sdk |
 oracle-java8-installer

The delta seems to be pkg-config. But it seems odd we should depend on that.

Reply:
@aiuto AFAIK when you install a `.deb` package via `dpkg -i`, the dependencies are not automatically installed (`dpkg` will simply complain about missing packages until you manually install them). Maybe it would be OK to just not install them, because the `apt-get install -f` should fix any missing dependencies afterwards, but it's been a while since I last used a Debian system, so I'm not sure.

@pranavj7Z It seems like the newer tool `apt` now allows one to install `.deb` packages from a downloaded file, too. This should work: `sudo apt install ./bazel-bazel_version-linux-x86_64.deb`. That way one supposedly doesn't have to manually install dependencies before and also the call to `apt-get install -f` is no longer needed. Could you try that and if it works, use that instead? Seems easier for users.